omod/src/main/webapp/resources/htmlforms/HMIS-ACP-018-HTSClientCard.xml

```diff
@@ -198,6 +214,15 @@
                     else
                         disable_fields(165159);
                 });
+<!--Hide fields for client initiated-->
+        jq("#hts-approach").change(function () {
```

Comment:
What does this code segment achieve? It does not seem to do anything.

Reply:
Removed